docroot/themes/humsci/su_humsci_theme/scss/utilities/mixins/_accessible-focus.scss

```diff
@@ -0,0 +1,11 @@
+@charset "UTF-8";
+
+//
+// Accessibile Focus
+// For applying an outline to the focus state of an element or component.
+// Helps improve accessibility.
+//
+
+@mixin accessible-focus {
```

Comment:
This seems like a good helper to add to decanter.

Reply:
Agreed! I've created a ticket in Decanter to add this there.